api/src/main/java/io/strimzi/api/kafka/model/KafkaConnectorSpec.java

```diff
@@ -42,38 +40,29 @@ public String getClassName() {
         return className;
     }
 
+    public void setClassName(String className) {
+        this.className = className;
+    }
+
     @Description("The maximum number of tasks for the Kafka Connector")
     @Minimum(1)
     public Integer getTasksMax() {
         return tasksMax;
     }
 
-    @Description("A list of topics to use as input for the Kafka Connector")
-    public List<String> getTopics() {
-        return topics;
+    public void setTasksMax(Integer tasksMax) {
+        this.tasksMax = tasksMax;
     }
 
-    @Description("The Config for the Kafka Connector Spec.")
-    public List<KafkaConnectorConfig> getConfig() {
+    @Description("The Config for the Kafka Connector Spec")
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    public Map<String, Object> getConfig() {
```

Comment:
Are you sure this should be `Map<String, Object>` and not just `Map<String, String>`?

Reply:
We were trying to find a similar declaration to have consistency and we found a Map<String, Object> config in some Spec files. Not a strong opinion really  @scholzj

Reply:
You are right. We are using it in other places for the sam epurpose as `<String, Object>` as well, so it shoud work. I will mark this as resolved.